### PR TITLE
add hook in productlot tooltip

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

File: htdocs/product/stock/product.php [hunk 1]
```diff
@@ -1132,6 +1132,7 @@
 				}
 				foreach ($details as $pdluo) {
 					$product_lot_static->id = $pdluo->lotid;
+					$product_lot_static->fk_product = $pdluo->fk_product_stock;
```
Comment (eldy) [inline htdocs/product/stock/product.php:1135]: fk_product_stock is a rowid in into llx_product_stock
but fk_product is an id of product . Are you sure ?

Comment (frederic34) [reply htdocs/product/stock/product.php]: i remove and will check later

**Comment by frederic34** [comment]
@eldy

**Comment by frederic34** [comment]
@eldy

**Pull request merged**